tests/Unit/Jobs/ScheduleStayInTouchTest.php

```diff
@@ -88,4 +88,35 @@ public function test_it_doesnt_dispatches_an_email_if_free_account()
             'stay_in_touch_trigger_date' => '2017-01-01 07:00:00',
         ]);
     }
+
+    public function test_it_reschedule_missed_stayintouch()
+    {
+        NotificationFacade::fake();
+
+        Carbon::setTestNow(Carbon::create(2019, 1, 1, 7, 0, 0, 'America/New_York'));
+
+        $account = factory(Account::class)->create([
+            'default_time_reminder_is_sent' => '07:00',
+            'has_access_to_paid_version_for_free' => 0,
+        ]);
+        $contact = factory(Contact::class)->create([
+            'account_id' => $account->id,
+            'stay_in_touch_trigger_date' => '2018-01-01 07:00:00',
+            'stay_in_touch_frequency' => 30,
+        ]);
+        $user = factory(User::class)->create([
+            'account_id' => $account->id,
+            'email' => 'john@doe.com',
+            'timezone' => 'America/New_York',
+        ]);
+
+        dispatch(new ScheduleStayInTouch($contact));
+
+        NotificationFacade::assertNotSentTo($user, StayInTouchEmail::class);
+        NotificationFacade::assertNothingSent();
+
+        $this->assertDatabaseHas('contacts', [
+            'stay_in_touch_trigger_date' => '2019-01-26 07:00:00',
```

Comment:
If the stay in touch frequency is set to 30, why is the date the 26th?

Reply:
because the frequency is not 'one month', it's '30 days' ...